coralproject/talk
# [CORL-1564] Add link to reply for flattened replies

## What does this PR do?

Shows a confirmation with a "Go to reply" when you make a reply and the reply level is flattened. This allows commenters to easily jump to their reply in a very long/nested thread of replies.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Enable flattened replies
- Make a comment that would appear at indent level 4 or higher (reply to a 3rd level indented coment)
- See the "Go to reply" confirmation and button link appear where the reply form used to be
- Click the "Go to reply"
- Stream iframe will scroll to the comment
